doc/ref_model/chapters/chapter07.md

```diff
@@ -507,3 +519,197 @@ Table 7-2 shows security capabilities
     tools
 -   Resiliency tests run (such as hardware failures or power failure
     tests)
+
+<a name="7.11"></a>
+## 7.11 Consolidated Security Requirements
+
+Please note that **Virtual Resource Manager (VRM)** is used to collectively represent VIM, K8s, …
+
+<a name="7.11.1"></a>
+### 7.11.1. System Hardening
+
+|  Ref | Requirement  |  Unit  | Definition  |
+|-------|------|------|-------|
+| sec.gen.001 | The Platform **must** maintain the state to what it is specified to be and does not change unless through change management process. |   |  |
+| sec.gen.002 | All systems part of NFVI **must** support password hardening (strength and rules for updates (process), storage and transmission, etc.) |  | Hardening: NIST SP 800-63B |
+| sec.gen.003 | All servers part of NFVI **must** support secure boot, root of trust and trusted boot |  | Secure boot - measured boot, verified boot  <br/> Root of trust: specified in RA or RI; should CNTT specify how? |
+| sec.gen.004 | The Operating Systems of all the servers part of NFVI **must** be hardened |  | NIST SP 800-123 |
+| sec.gen.005 | The Platform **must** support Operating System level access control |  | Details on OS |
+| sec.gen.006 | The Platform **must** support Secure logging |  | Details |
+| sec.gen.007 | All servers part of NFVI **must** be Time synchronized with authenticated Time service |  | |
+| sec.gen.008 | All servers part of NFVI **must** be regularly updated to address security vulnerabilities |  | |
+| sec.gen.009 | The Platform **must** support Software integrity protection and verification |  | |
+| sec.gen.010 | The NFVI **must** support Secure storage (all types) |  | Expand/Delete based on other reqts |
+| sec.gen.011 | The NFVI **should** support Read and Write only storage partitions (write only permission to one or more authorized actors) |  | NIST reference? |
+| sec.gen.012 | The Platform **must** ensure that only authorized actors have physical access to the underlying infrastructure. |  | |
```

Comment:
How can a Platfrm ensure this?

Reply:
@CsatariGergely Platform = NFVI + VRM (such as VIM). Please suggest alternate language if the explanation is unsatisfactory.

Reply:
The platform does not encompass the building or facilities, server cages, etc needed to deny physical access to the servers.  This is a valid datacenter requirement, but it should not be a CNTT requirement.  I suggest removing sec.gen.012 as it is out of scope for CNTT and thus unenforceable.
```suggestion
```

Reply:
I think sec.gen.012 is a datacenter requirement, not a platform requirement.  If the platform is limited to the servers, storage, networking, and software used to create the cloud infrastructure it doesn't incorporate physical datacenter design aspects (like physical security).  A vendor NFVI cannot be deemed to comply with this requirement as it needs to be installed in a physical location that is not of the vendor's choosing.
Can this requirement be removed?  I know it's a valid requirement from a holistic perspective, but when considering the Cloud Infrastructure, it's not in that area of focus and thus should be specified in other documents but not CNTT ones.